app/controllers/simulator_controller.rb

```diff
@@ -94,6 +95,12 @@ def verilog_cv
     render json: response.to_s, status: response.code
   end
 
+  def allow_iframe_lti
```

Comment:
Seems to be repeated in 3 places, can it be refactored?

Reply:
will try to refractor in next PR